configure.ac

```diff
@@ -2305,6 +2305,69 @@ then
 fi
 # }}}
 
+# --with-libdpdk {{{
+AC_ARG_WITH(libdpdk, [AS_HELP_STRING([--with-libdpdk@<:@=PREFIX@:>@], [Path to the DPDK build directory.])],
+[
+  if test "x$withval" != "xno" && test "x$withval" != "xyes"
+	then
+		with_dpdk_path="$withval"
+		with_libdpdk="yes"
+	else
+		if test "x$withval" = "xno"
+		then
+			with_libdpdk="no (disabled)"
+		fi
+	fi
+], [with_libdpdk="no"])
+
+if test "x$with_libdpdk" = "xyes"
+then
+	RTE_BUILD="$with_dpdk_path"
+	DPDK_INCLUDE="$RTE_BUILD/include/dpdk"
+	DPDK_LIB_DIR="$RTE_BUILD/lib"
+	DPDK_LIB="-ldpdk"
```

Comment:
You're setting DPDK_LIB here, and add it to LIBS, but further down you use AC_CHECK_LIB to check in libdpdk, so this is not needed.